pages/report-repair/[route].js

```diff
@@ -288,6 +289,20 @@ function ReportRepair() {
           ]}
         />
       )
+    case 'outside-door-problems':
+      return (
+        <RepairProblemBestDescription
+          handleChange={handleChange}
+          values={values}
+          options = {[
+            { value: 'shedDoor', title: 'Shed door'},
+            { value: 'outhouseCupboardDoor', title: 'Outhouse cupboard door'},
+            { value: 'woodenBackDoor', title: 'Wooden back door'},
+            { value: 'UPVCBackDoor', title: 'UPVC back door'},
```

Comment:
To stick with the naming convention:
```suggestion
            { value: 'upvcBackDoor', title: 'UPVC back door'},
```